src/librustc/hir/map/mod.rs

```diff
@@ -310,14 +310,14 @@ impl<'hir> Map<'hir> {
         self.definitions.as_local_node_id(def_id.to_def_id()).unwrap()
     }
 
-    pub fn describe_def(&self, node_id: NodeId) -> Option<Def> {
+    fn def_kind(&self, node_id: NodeId) -> Option<DefKind> {
```

Comment:
I was surprised that this worked and users of `describe_def` never needed the `DefId`-less variants, but if it's called *only* from a `DefId`-based query, then it's understandable.